modules/mailer/mailer.go

```diff
@@ -122,11 +122,10 @@ func (s *smtpSender) Send(from string, to []string, msg io.WriterTo) error {
 	}
 	defer conn.Close()
 
-	isSecureConn := false
+	isSecureConn := opts.IsTLSEnabled
 	// Start TLS directly if the port ends with 465 (SMTPS protocol)
-	if strings.HasSuffix(port, "465") {
```

Comment:
Perhaps have the logic be
```
isSecureConn =  (port =465) || TLSEnabled
```
for backwards compatibility per @lafriks suggestion?